OPHD/States/MapViewStateTurn.cpp

```diff
@@ -351,28 +351,18 @@ void MapViewState::updateBiowasteRecycling()
 	auto residenceIterator = residences.begin();
 	for (auto recyclingFacility : recyclingFacilities)
 	{
-		int count = 0;
-		
 		if (!recyclingFacility->operational()) { continue; } // Consider a different control structure
 
 		Recycling* recycling = static_cast<Recycling*>(recyclingFacility);
-
-		for (; residenceIterator != residences.end(); ++residenceIterator)
+		for (int count = 0; count < recycling->residentialSupportCount(); ++count)
 		{
+			if (residenceIterator == residences.end()) {
+				return; // No more residences, so don't waste time iterating over remaining recycling facilities
```

Comment:
Should either move the brace placement for consistency or make this a single line with the comment above it.

Reply:
Oh right, I meant to match up the brace style. Guess I got distracted on that one.